docs/architecture/cloud-native/candidate-apps.md

```diff
@@ -1,52 +1,57 @@
 ---
 title: Candidate Apps for Cloud Native
-description: Architecting Cloud Native .NET Apps for Azure | Candidate Apps for Cloud Native
-ms.date: 06/30/2019
+description: Architecting Cloud-Native .NET Apps for Azure | Candidate Apps for Cloud Native
```

Comment:
The description is not correct here and probably in the other articles too. It's using the microservices model which was done wrong and we could never fixed it because of lack of resources and time. 
The description shows up in search engines so it should describe what the article is about, kind of inviting users to read it and why they'd want to click that link.

Reply:
Bit confused. Are we saying to add more verbiage to the description?

Reply:
Not exactly, It should be replaced with a more proper description. The serverless archicture book was one that we followed the SEO guidelines, for example;
https://github.com/dotnet/docs/blob/master/docs/architecture/serverless/serverless-architecture-considerations.md